### ISSUE TITLE
How to save a model with the smallest loss during training (epochs)

### ISSUE THREAD

**Issue by GMSL1706** [airctic/icevision#1085]
Hello,

I use fastai for training according to [this example](https://airctic.com/0.12.0/mmdet_custom_config/#training-using-fastai). As can be seen with the evolution of the loss through the epochs, the best loss is often before the last epoch. Question: When I save the model using `torch.save(model.state_dict(), "path/model_name.pth")`, does it save the model with best loss or with always latest epoch?

If it's always at the last epoch, how to save the model with always the best loss?

**Comment by FraPochetti** [comment]
> with always latest epoch?

Always latest epoch 

If you want to save the model at the best epoch you need to use the `SaveModelCallback` [callback](https://docs.fast.ai/callback.tracker.html#SaveModelCallback)
